src/fidesops/ops/graph/analytics_events.py

```diff
@@ -98,7 +98,7 @@ def failed_graph_analytics_event(
         docker=in_docker_container(),
         event="privacy_request_execution_failure",
         event_created_at=datetime.now(tz=timezone.utc),
-        local_host=False,  # Temporarily defaulting to False, but we are going to change this to None
+        local_host=None,
         endpoint=None,
```

Comment:
Verified locally that it is now possible to make these calls to fideslog after Phil's work here https://github.com/ethyca/fideslog/pull/79